django2pydantic/handlers/json.py

```
@@ -1,15 +1,28 @@
 
 from django.db import models
-from pydantic import Json
+from pydantic import BeforeValidator, Json
 
 from django2pydantic.handlers.base import DjangoFieldHandler
-from django2pydantic.types import GetType, SetType
+from django2pydantic.types import GetType, SetType, SupportedPydanticTypes
 
 
-class JSONFieldHandler(DjangoFieldHandler[models.JSONField[SetType, GetType]]):
+def ensure_json_str(value: Any) -> str:  # pyright: ignore [reportExplicitAny]
```

Ruff (0.11.9)

Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)